staging/src/k8s.io/client-go/examples/third-party-resources/types.go

```diff
@@ -23,6 +23,18 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
+const (
+	ExampleDomain              = "k8s.io"
```

Comment:
it's not a good example to have resources under k8s.io. Can we move that to some sub-domain?